README.md

```diff
@@ -410,13 +423,13 @@ To send a multipart request, `:multipart` may be supplied as a sequence of maps
 the `:body` of the request with an `InputStream` of the supplied parts.
 
 ```clojure
-(hc/get "http://moo.com"
-        {:multipart [{:name "title" :content "My Awesome Picture"}
-                     {:name "Content/type" :content "image/jpeg"}
-                     {:name "foo.txt" :part-name "eggplant" :content "Eggplants"}
-                     {:name "file" :content (io/file ".test-data")}
-                     {:name "data" :content (.getBytes "hi" "UTF-8") :content-type "text/plain" :file-name "data.txt"}
-                     {:name "jsonParam" :content (io/file ".test-data") :content-type "application/json" :file-name "data.json"}]})
+(hc/post "http://moo.com"
```

Comment:
Just making a quick doc change here. Doesn't make sense to do a multipart get request :).